Integration/FRAIHMWORK Integrator Starter Kit Collection.json

```diff
@@ -0,0 +1,2038 @@
+												"{{baseUrl}}{{Mitigation}}"
+											],
+											"path": [
+												"mitigation",
+												"state",
+												"ACCEPTED"
+											]
+										},
+										"description": "This endpoint allows requesting services to find information about registered instances that FRAIHMWORK is currently aware of"
+									},
+									"response": []
+								}
+							]
+						}
+					]
+				},
+				{
+					"name": "/subscription/mitigation",
+					"item": [
+						{
+							"name": "POST",
+							"item": [
+								{
+									"name": "Subscribe to mitigation",
+									"event": [
+										{
+											"listen": "prerequest",
+											"script": {
+												"exec": [
+													"pm.collectionVariables.set(\"expirationTime\",\"null\");\r",
+													"var now = new Date();\r",
+													"var utc = new Date(now.getTime() + now.getTimezoneOffset() * 60000);\r",
+													"utc.setMinutes(utc.getMinutes() + 30 );\r",
+													"pm.collectionVariables.set(\"expirationTime\", utc);"
+												],
+												"type": "text/javascript"
+											}
+										},
+										{
+											"listen": "test",
+											"script": {
+												"exec": [
+													"pm.collectionVariables.set(\"UUIDmitigationSubscription\", \"null\");\r",
+													"var mitigatorJsonData = pm.response.json();\r",
+													"pm.collectionVariables.set(\"UUIDmitigationSubscription\", mitigatorJsonData.itemId);\r",
+													"\r",
+													"pm.test(\"Successful Mitigator Subscription POST\", function () {\r",
+													"    pm.response.to.have.status(201);\r",
+													"});\r",
+													""
+												],
+												"type": "text/javascript"
+											}
+										}
+									],
+									"request": {
+										"auth": {
+											"type": "bearer",
+											"bearer": [
+												{
+													"key": "token",
+													"value": "{{access_token}}",
+													"type": "string"
+												}
+											]
+										},
+										"method": "POST",
+										"header": [
+											{
+												"key": "Content-Type",
+												"value": "application/json"
+											}
+										],
+										"body": {
+											"mode": "raw",
+											"raw": "{\n    \"baseUrl\": \"{{$randomUrl}}\",\n    \"interestedUuids\": [\n        \"{{UUIDMitigationResponse}}\"\n    ],\n    \"stopTime\": \"{{expirationTime}}\"\n}"
+										},
+										"url": {
+											"raw": "{{baseUrl}}{{Mitigation}}/subscription/mitigation",
+											"host": [
+												"{{baseUrl}}{{Mitigation}}"
+											],
+											"path": [
+												"subscription",
+												"mitigation"
+											]
+										},
+										"description": "Allows a system to report one or more faults currently affecting the system. Using this endpoint means that the client is not managing their own fault IDs, and the fault will require a new tracking UUID (that will be created and assigned in the response)."
```

Comment:
Copy paste description. Should be corrected

Reply:
This is the last one I'm commenting on. This whole file needs to be scrubbed for the documentation sections.